spec/features/user_registration_spec.rb

```diff
@@ -1,54 +1,55 @@
 require 'rails_helper'
 
-describe "signup page" do
-  it "should have the content 'User Registration'" do
-    visit '/new_user'
-    page.should have_content ('User Registration')
+  describe "signup page" do
+    it "should have the content 'Sign up'" do
+      visit new_user_registration_path
+      expect(page).to have_content('Sign up')
+    end
+  end
+  
+
+  describe "sign up user" do
+    it "new account" do
+      visit new_user_registration_path
+        fill_in 'Username', with: 'username'
+        fill_in 'Email', with: 'user@example.com'
+        fill_in 'Password', with: 'password'
+        fill_in 'Password confirmation', with: 'password'
+        click_button 'Sign up'
+
+        expect(current_path).to eq root_path
+        expect(page).to have_content('activate')
+    end
   end
-end
 
-describe "signup flow" do
+  describe "authenticate" do
+  #  before do 
```

Comment:
You need to wrap 27 to 35 in an it block. That will fix the error you are receiving.